packages/flutter/lib/src/animation/listener_helpers.dart

```diff
@@ -38,7 +40,8 @@ abstract class EagerListenerMixin implements _ListenerMixin {
   void dispose();
 }
 
-abstract class LocalAnimationListenersMixin extends _ListenerMixin {
+/// A mixin that maintains a list of listeners in a member variable.
```

Comment:
"A mixin that implements the addListener/removeListener protocol and notifies all the registered listeners when notifyListeners() is invoked."